image/templates/helm/shared/templates/02-scanner-v4-08-indexer-service.yaml

```diff
@@ -0,0 +1,48 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.scannerV4._indexerEnabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: scanner-v4-indexer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "service" "scanner-v4-indexer") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "service" "scanner-v4-indexer") | nindent 4 }}
+spec:
+  selector:
+    app: scanner-v4-indexer
+  clusterIP: None
+  ports:
+  - name: grpcs-scanner-v4-indexer
```

Comment:
I think this should just be `grpc-scanner-v4-indexer` (drop the `s` for `grpc`). The purpose of the prefix is for Istio I believe: https://istio.io/v1.0/docs/setup/kubernetes/spec-requirements/

Reply:
thanks